rllib/tests/run_regression_tests.py

```diff
@@ -139,16 +139,6 @@
             print(f"Skipping framework='{args.framework}' for QMIX.")
             continue
 
-        # Always run with eager-tracing when framework=tf2 if not in local-mode.
-        # Ignore this if the yaml explicitly tells us to disable eager tracing
-        if (
-            args.framework == "tf2"
-            and not args.local_mode
-            and not exp["config"].get("eager_tracing") is False
-        ):
-
-            exp["config"]["eager_tracing"] = True
```

Comment:
Can we actually make this so that when local mode is True for the regression tests script (we use this only for debugging, right?), eager tracing will be False?

Reply:
I think in almost all cases we are happy that eager tracing is disabled when using local mode here.

Reply:
True, but I feel like then you should just set it in your config, no? It's not good to change stuff b/c we assume something w/o the user being in control. I have run into this issue several times while debugging, thinking that eager tracing was True (I wanted to debug a bug that only happened for eager_tracing=True), when it wasn't b/c I was also using local mode. It took me a while to find out that RLlib had sneakily changed my config :)